addons/project/static/src/scss/project_form.scss

```diff
@@ -37,3 +37,9 @@
 .o_data_cell .o_favorite i.fa {
     font-size: 1.35em;
 }
+
+.o_kanban_quick_create .o_field_widget .o_input  {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
```

Comment:
Zut (I did not see it before, sorry...)

Is it really needed? 🤔
Because that selector is generic and so it will impact more than our kanban view. 🙈

Reply:
Mmmh so this is the second specs cropping placeholders in the quickcreate en kanban view
But yes it's better to just make it for our scope